src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -128,6 +128,86 @@ class Target
             await VerifyAnalyzerAsyncCs(source, tfmAndOption);
         }
 
+        [Fact]
+        public async Task ProjectWithPlatformNeutralAssemblyPropertyTest()
+        {
+            var source = @"
+using System.Collections.Generic;
+using System.Runtime.Versioning;
+
+[assembly:SupportedOSPlatform(""linux"")] 
+public class Test
+{
+    private string program;
+
+    [SupportedOSPlatform(""windows"")] // Can overwrite parent browser support as cross platform
```

Comment:
> Can overwrite parent browser support as cross platform

Is the parent browser or linux here? Was that a typo?

Reply:
yeah that is typo, initially i used browser, later changed to linux to repro a bug found in runtime test